.travis.yml

```diff
@@ -1,6 +1,17 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - node    # latest
+  - 8
   - 6
-cache: yarn
-script: travis_wait 15 yarn test   # Increase "no output received" timeout
+
+cache:
+  yarn: true
+  directories:
+    - ".eslintcache"
+    - "node_modules"
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
This should only be needed for a specific version, no? Or Is Travis lacking behind in the Yarn version it supports?

Reply:
Travis has 1.3.2, this script updates it to the latest.